src/main/java/org/opensearch/geospatial/index/mapper/xypoint/XYPointFieldMapper.java

```diff
@@ -115,19 +122,25 @@ public XYPointFieldMapper build(
             ParsedPoint nullValue,
             CopyTo copyTo
         ) {
+            var processor = new XYPointQueryProcessor();
```

Comment:
either use var at all the places or use specific type at all the places.

Reply:
ack